CHANGELOG.adoc

```diff
@@ -13,6 +13,10 @@ For a detailed view of what has changed, refer to the {uri-repo}/commits/main[co
 
 == Unreleased
 
+Documentation::
+
+  * add id's for all parameters of process-asciidoc, to be able to generate direct urls (https://github.com/uniqueck[@uniqueck]) (#533)
```

Comment:
Capitalize A in add?

Reply:
Also, you updated all configurations for all goals, not only `process-asciidoc` :+1: that deserves some credit

Reply:
You are so right ;)